### PR TITLE
Qt: show current calibration values

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Let's just do this now: https://github.com/commaai/openpilot/blob/master/selfdrive/locationd/calibrationd.py#L83. It should simplify the code in calibrationd and UI.

**Comment by pd0wm** [comment]
I don't see the UI reading the param anywhere when we're offroad

**Comment by deanlee** [comment]
Now read calibration from `CalibrationParams` at startup

File: selfdrive/ui/qt/offroad/settings.cc [hunk 1]
```diff
@@ -93,8 +93,23 @@ DevicePanel::DevicePanel(QWidget* parent) : QWidget(parent) {
                                       GLWindow::ui_state.scene.driver_view = true; }
                                     ));
 
-  offroad_btns.append(new ButtonControl("Reset Calibration", "RESET",
-                                   "openpilot requires the device to be mounted within 4° left or right and within 5° up or down. openpilot is continuously calibrating, resetting is rarely required.", [=]() {
+  QString resetCalibDesc = "openpilot requires the device to be mounted within 4° left or right and within 5° up or down. openpilot is continuously calibrating, resetting is rarely required.";
+  if (std::string params = Params().get("CalibrationParams"); !params.empty()) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/settings.cc:97]: move reading the param outside of the if statement

**Comment by adeebshihadeh** [comment]
Looks good, but it doesn't update now.

**Pull request merged**